nvflare/private/fed/server/job_cmds.py

```diff
@@ -165,12 +166,17 @@ def get_spec(self):
         )
 
     def authorize_job_file(self, conn: Connection, args: List[str]):
-        if len(args) < 2:
+        """
+        Args: cmd_name tx_id job_id file_name [end]
+        """
+        if len(args) < 3:
```

Comment:
This needs to be len(args) <4? 
Since we need "cmd_name tx_id job_id file_name"
And we need to change the text in the CommandSpec up there in Line 159.
`usage=f"{AdminCommandNames.DOWNLOAD_JOB_FILE} tx_id job_id file_name",`

Reply:
Should be 4, yes. Will fix.

Reply:
Fixed.